shelley/chain-and-ledger/shelley-spec-ledger-test/test/Test/Shelley/Spec/Ledger/Examples/Mir.hs

```diff
@@ -157,8 +157,8 @@ insufficientMIRWits = mirWits [0 .. 3]
 
 txEx1 ::
   forall c.
-  (Mock (Crypto (ShelleyEra c))) =>
-  [KeyPair 'Witness (Crypto (ShelleyEra c))] ->
+  (Mock c)=> -- (Crypto (ShelleyEra c))) =>
```

Comment:
Dead comment? Also weird formatting

Reply:
Not sure how this came about. But note that  (Crypto (ShelleyEra c)) is the same as 'c' by definition.
Any way the comment is removed and the formatting made more standard.